pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -547,6 +549,21 @@ func TestScaleBounds(t *testing.T) {
 		min:     "1",
 		wantMin: 1,
 		wantMax: 0,
+	}, {
+		name:    "only min, overriden",
+		min:     "1",
+		wantMin: 1,
+		wantMax: 10,
+		config: &autoscalerconfig.Config{
+			DefaultMaxScale: 10,
+		},
+	}, {
+		name:    "max and config",
```

Comment:
can we add a test for max and config present but max is non-zero?

Reply:
i replaced because i'm not sure the case where both are defined and max is zero is separately useful. I'll add back if others disagree.